pom.xml

```diff
@@ -520,6 +526,14 @@
     </dependencies>
   </dependencyManagement>
 
+  <dependencies>
+    <dependency>
+      <groupId>com.google.errorprone</groupId>
+      <artifactId>error_prone_core</artifactId>
+      <scope>provided</scope>
```

Comment:
Shall we move `scope` to management section to align with `error_prone_core`?

Reply:
I'm not a fan of having the `scope` in the management section, it's too far removed from the use-site and seems all too prone to make mistakes. The way we put `scope` in the management section has been disturbing me for a long time. I'd rather not have it in the management section at all.

Reply:
Either way I think. As long as it is consistent. :D